documentation/content/installanduninstall.rst

```diff
@@ -13,34 +13,25 @@
 
 Package Dependencies
 --------------------
-You have to figure out the correct package name in your distro for the
-following dependencies:
+We currently support *arch* and *debian* for automatic dependencies
+installation. These are the current dependencies for *arch*:
 
-* libguestfs
-* Qemu
-* Bash >= 4.4
-* git
-* tar
-* python-docutils
-* rsync
+.. include:: ../dependencies/arch.dependencies
+   :literal:
```

Comment:
Is it possible to make these dependencies be displayed in a single line in the documentation? This will make it easier for someone to just copy and paste in the terminal.

Reply:
Yes, without the `:literal:` that is the behavior, I just thought this was easier to read. But a single line would be easier for copying.